test/unit-test/ota_utest.c

```diff
@@ -1727,6 +1755,55 @@ void test_OTA_ReceiveLastFileBlockTooSmall()
     TEST_ASSERT_EQUAL( OtaAgentStateWaitingForJob, OTA_GetState() );
 }
 
+void test_OTA_ReceiveWithNullFile()
```

Comment:
could you add a comment with what happens when you receive a null file pointer?

Reply:
done